portality/scripts/anon_import.py

```diff
@@ -0,0 +1,94 @@
+import esprit, codecs, json
+from portality.core import app, initialise_index
+from portality.store import StoreFactory
+from portality import constants
+
+
+def do_import(config):
+    host = app.config["ELASTIC_SEARCH_HOST"]
+    index = app.config["ELASTIC_SEARCH_DB"]
+    if config.get("elastic_search_host") is not None:
+        host = config.get("elastic_search_host")
+        app.config["ELASTIC_SEARCH_HOST"] = host
+    if config.get("elastic_search_db") is not None:
+        index = config.get("elastic_search_db")
+        app.config["ELASTIC_SEARCH_DB"] = index
+
+    print("\n")
```

Comment:
Unfortunately not.  This print statement adds an extra line after the block of print statements before it.  Since print() implicitly includes a newline, if we also added it to the block before it, everything would have an extra line between them, and there wouldn't be a differentiating bit of whitespace between that block and the next.

I'm not sure this is wrong, and is just personal preference :)